### PR TITLE
Add some Russian additives, ingredients and minerals

### PR DESCRIPTION
**Pull request by blazern**
Adding some Russian additives, ingredients and minerals.

### CODE REVIEW

File: taxonomies/additives.txt [hunk 1]
```diff
@@ -1487,7 +1487,7 @@ oc:E150, caramèl
 pl:E150, karmel
 pt:E150, caramelo
 ro:E150, Carmel
-ru:E150, карамель
+ru:E150, карамель, burnt sucrose
```
Comment (stephanegigandet) [inline taxonomies/additives.txt:1490]: the translations should only be in the target language, why add "burnt sucrose"?
and it looks like we have no product with it: https://world.openfoodfacts.org/ingredient/burnt-sucrose

have you seen it in an actual product?

Comment (blazern) [reply taxonomies/additives.txt:1490]: Oops, looks like a mistake

**Comment by blazern** [comment]
Removed `burnt sucrose`.

**Review by stephanegigandet — APPROVED** [review]

**Comment by blazern** [comment]
Resolved a conflict

**Pull request merged**